src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const box = document.querySelector('ul');
+
+function getNumber(string) {
+  return +string.dataset.salary.split('$').join('').split(',').join('');
+}
+
+function sortList(data) {
+  const sortedData = [...data].sort((a, b) => getNumber(b) - getNumber(a));
+
+  box.append(...sortedData);
+}
+
+function getEmployees(employees) {
+  const arrayOfEmployees = [...employees].map(employee => {
```

Comment:
does map mutate original array?
```suggestion
  const arrayOfEmployees = employees.map(employee => {
```